ui/grid_menu.py

```diff
@@ -0,0 +1,107 @@
+from time import sleep
+
+from base_ui import BaseUIElement
+from canvas import Canvas
+
+class GridMenu(BaseUIElement):
+
+	GRID_WIDTH = 3
```

Comment:
be able to define it as an init() argument? seems like a good base UI element that we could later move, say, `DatePicker` to, I will think about it more.